kornia/core/check.py

```diff
@@ -113,11 +114,12 @@ def KORNIA_CHECK_TYPE(x, typ, msg: Optional[str] = None):
     Example:
         >>> KORNIA_CHECK_TYPE("foo", str, "Invalid string")
     """
+    # TODO: Move to use typeguard here dropping support for JIT
```

Comment:
I will just push this in another PR because JIT will be mad with any typing on these `Check` functions -- So, basically, will drop all functions JIT support for functions that use `kornia.core.check`

Reply:
#2219